src/inausoft.netCLI/Deserialization/IArgumentDeserializer.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace inausoft.netCLI.Deserialization
+{
+    public interface IArgumentDeserializer
```

Comment:
`Argument` or `Option`? We need to agreed to single naming convention.

Reply:
Let's go with the 'Option' - I will rename it to `IOptionsDeserializer`.